common/enableEpochs/flags.go

```diff
@@ -504,3 +508,13 @@ func (fh *flagsHolder) IsHeartbeatDisableFlagEnabled() bool {
 func (fh *flagsHolder) IsMiniBlockPartialExecutionFlagEnabled() bool {
 	return fh.isMiniBlockPartialExecutionFlag.IsSet()
 }
+
+// IsManagedCryptoAPIsFlagEnabled returns true if managedCryptoAPIsFlag is enabled
+func (fh *flagsHolder) IsManagedCryptoAPIsFlagEnabled() bool {
```

Comment:
shouldn't we use these flags in the VM factory?

Reply:
that's right, but enableEpochsHandler will be integrated into arwen as well